src/aws.ts

```diff
@@ -354,7 +356,7 @@ export namespace $AWS {
       },
       unhandledContext(kind, context) {
         throw new Error(
-          `$AWS.${kind} is only available within an '${ASL.ContextName}' context, but was called from within a '${context.kind}' context.`
+          `$AWS.${kind} is only available within an '${ASL.ContextName}' context, but was called from within a '${context}' context.`
```

Comment:
It is now. In the case of an error, I think we only care about the kind. This change allowed me to make the integration types into objects instead of functions

 `integrationImpl.appSyncVtl.request(call, context)`